listen_to_mqtt.sh

```diff
@@ -66,126 +71,157 @@ listen_to_mqtt() {
           ;;
         charging-start)
-          log_notice "Start Charging"
-          send_command $vin $msg
+          send_command $vin $msg "Start charging"
           ;;
         charging-stop)
-          log_notice "Stop Charging"
-          send_command $vin $msg
+          send_command $vin $msg "Stop charging"
           ;;
-        charge-port-open)
-          log_notice "Open Charge Port"
-          send_command $vin $msg
-          ;;
-        charge-port-close)
-          log_notice "Close Charge Port"
-          send_command $vin $msg
+        climate-off)
+          send_command $vin $msg "Turn off climate control"
           ;;
         climate-on)
-          log_notice "Start Climate"
-          send_command $vin $msg
+          send_command $vin $msg "Turn on climate control"
           ;;
-        climate-off)
-          log_notice "Stop Climate"
-          send_command $vin $msg
+        drive)
+          send_command $vin $msg "Remote start vehicle"
           ;;
         flash-lights)
-          log_notice "Flash Lights"
-          send_command $vin $msg
+          send_command $vin $msg "Flash lights"
           ;;
         frunk-open)
-          log_notice "Open Frunk"
-          send_command $vin $msg
+          send_command $vin $msg "Open vehicle frunk"
           ;;
         honk)
-          log_notice "Honk Horn"
-          send_command $vin $msg
+          send_command $vin $msg "Honk horn"
+          ;;
+        list-keys)
+          send_command $vin $msg "List public keys enrolled on vehicle"
           ;;
         lock)
-          log_notice "Lock Car"
-          send_command $vin $msg
+          send_command $vin $msg "Lock vehicle"
+          ;;
+        media-toggle-playback)
+          send_command $vin $msg "Toggle between play/pause"
+          ;;
+        ping)
+          send_command $vin $msg "Ping vehicle"
+          ;;
+        software-update-cancel)
+          send_command $vin $msg "Cancel a pending software update"
+          ;;
+        software-update-start)
+          send_command $vin $msg "Start software update after delay"
+          ;;
+        tonneau-close)
+          send_command $vin $msg "Close Cybertruck tonneau"
+          ;;
+        tonneau-open)
+          send_command $vin $msg "Open Cybertruck tonneau"
+          ;;
+        tonneau-stop)
+          send_command $vin $msg "Stop moving Cybertruck tonneau"
+          ;;
+        trunk-close)
+          send_command $vin $msg "Close vehicle trunk"
+          ;;
+        trunk-move)
+          send_command $vin $msg "Toggle trunk open/closed"
+          ;;
+        trunk-open)
+          send_command $vin $msg "Open vehicle trunk"
           ;;
         unlock)
-          log_notice "Unlock Car"
-          send_command $vin $msg
+          send_command $vin $msg "Unlock vehicle"
+          ;;
+        wake)
+          send_command $vin " "-domain vcsec $msg" "Wake up vehicule"
```

Comment:
extra quote I think + typo: "vehicule" --> "vehicle"

Reply:
Yes found that out when I tested as an HA add-on .. you have good eyes :)

Reply:
Done.